documentation/slate/source/includes/realtime.md

```diff
@@ -12,6 +12,7 @@ The effect of a disruption can be one of the following:
 	<li>[SIGNIFICANT_DELAYS](#SIGNIFICANT_DELAYS)</li>
 	<li>[REDUCED_SERVICE](#REDUCED_SERVICE)</li>
 	<li>[MODIFIED_SERVICE](#MODIFIED_SERVICE)</li>
+    <li>[ADDITIONAL_SERVICE](#ADDITIONAL_SERVICE)</li>
```

Comment:
minor: weird indent

Reply:
Fixed. Though it didn't change anything once built